pkg/cmd/admin/router/router.go

```diff
@@ -158,7 +166,17 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 		}
 	}
 
-	ports, err := app.ContainerPortsFromString(cfg.Ports)
+	// For host networking, use the default ports 80/443. We don't have
+	// config options as yet to specify what the externally exposed
+	// ports are.
+	ports, err := app.ContainerPortsFromString(defaultPorts)
```

Comment:
I don't understand the reasoning of forcing it to use defaultPorts in the case of host networking.  If the config is defaulted to defaultPorts and the user passes something in shouldn't it always override?  If they don't then it will be == defaultPorts anyway.